ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -114,7 +114,7 @@
     - name: Remove remaining files
       file: path={{ item }} state=absent
       with_items:
-        - "~{{ ansible_ssh_user }}/.kube"
+        - "{{ ansible_env.HOME }}/.kube"
```

Comment:
Got the same feeling :). Reading through stackoverflow articles and ansible docs and can't find anything particular and usable for the ``ansible_env.HOME`` case. So far have not found anything saying it is incorrect usage. With the ``ansible_env.HOME`` it is explicitly mentioned I want remote host directory which corresponds to a user under which the ansible is ssh'ed.